utils/setup_for_python_metadata.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import setup
```

Comment:
We don't need setuptools for this, we can use `distutils.core`, since we have no setuptools dependency definitions.

Cf. https://docs.python.org/3/distutils/introduction.html

Reply:
I have updated it to use only `distutils.core`, this way it generates only one egg-info file instead of a directory so I have also updated the specfile.